### PR TITLE
adjusting curated response format, notification of raw format depreci…

### PR DESCRIPTION
**Pull request by zikalino**
…ation.

##### SUMMARY
This is just final adjustment to VMSS curated return format added notification regarding depreciation of "format" option in 2.10 as well.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
azure_rm_virtualmachine_scaleset_facts

##### ANSIBLE VERSION
2.6

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py [hunk 1]
```diff
@@ -70,7 +71,7 @@
 '''
 
 RETURN = '''
-azure_vmss:
+vmss:
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py:74]: User will get break if this field name is modifies.

**Comment by Fred-sun** [comment]
@zikalino Could you help check the comment? The PR may need to be updated?

**Comment by Fred-sun** [comment]
@zikalino Could you help update the comments? Thanks!

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py (hunk 1 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py:74]: @yuwzho only the doc is modified, the module will return both formats till version 2.10.

**Review by yungezz — APPROVED** [review]

**Pull request merged**